From c25e20cbe74ca474a299f84b1a858c076c6b3e92 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Mon, 26 May 2025 14:42:59 +0300
Subject: [PATCH] [SW-229991] gaudi3: common_utils: fix max nodes

Fix max nodes for Skyriver and Whitebox.
Skyriver have 2 host x 4 nodes = 8 max nodes
Whitebox have 16 host x 1 node = 16 max nodes

Signed-off-by: Guy Nisan <gnisan@habana.ai>
Change-Id: I1d9071098af908eef338eec3d26d86ebe648d104
---

diff --git a/drivers/misc/common_utils/gaudi/gaudi3/common_utils.c b/drivers/misc/common_utils/gaudi/gaudi3/common_utils.c
index 0dd4313..06e75ae 100644
--- a/drivers/misc/common_utils/gaudi/gaudi3/common_utils.c
+++ b/drivers/misc/common_utils/gaudi/gaudi3/common_utils.c
@@ -642,8 +642,8 @@
 	return 0;
 }
 
-#define GAUDI3_RACK_SERDES_NODES_MAX_NUM		4
-#define GAUDI3_RACK_WHITEBOX_SERDES_NODES_MAX_NUM	8
+#define GAUDI3_RACK_SERDES_NODES_MAX_NUM		8
+#define GAUDI3_RACK_WHITEBOX_SERDES_NODES_MAX_NUM	16
 
 uint8_t get_max_number_of_nodes(uint32_t link_id)
 {
